packages/launchpad/cypress/e2e/choose-a-browser.cy.ts

```diff
@@ -59,9 +53,9 @@ describe('Choose a Browser Page', () => {
 
       cy.get('[data-cy="alert-header"]').should('contain', 'Warning: Browser Not Found')
       cy.get('[data-cy="alert-body"]')
-      .should('contain', 'We could not identify a known browser at the path you specified: /path/does/not/exist')
+      .should('contain', 'We could not identify a known browser at the path you provided: /path/does/not/exist')
+      .should('contain', 'spawn /path/does/not/exist ENOENT')
       .validateExternalLink({
-        name: 'how to troubleshoot launching browsers',
         href: 'https://on.cypress.io/troubleshooting-launching-browsers',
```

Comment:
Was curious about these changes, but the message just changed. The links are presented as-is now, without overriding text :+1:

Reply:
Yeah @brian-mann removed the `delinkify` helper, we can add it back in but I think he had some concerns with it or something.

Reply:
yeah I want to review that as part of the next PR - which works on the markdown rendering in 10.x